src/queries/pet/updatePetQuery.ts

```diff
@@ -0,0 +1,17 @@
+import { type IPet } from '../../interfaces/fakeDataTypes'
+import { Pet } from '../../models'
+
+const updatePetQuery = async (postId: number, petData: IPet): Promise<IPet | null> => {
+  const [count, [updatedPet]] = await Pet.update(
```

Comment:
I remember that this should work, try it and tell me
```suggestion
  const [count, updatedPet] = await Pet.update(
```